### PR TITLE
Add Nested Route Example to README.md

### PR DESCRIPTION
**Pull request by 38elements**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,17 +44,22 @@ If there is an error rendering the destination route, a 404 will be displayed.
...
 
-Defining what component(s) to load for a given URL is easy and declarative.
-You can even mix-and-match URL parameters and normal `props`.
-You can also make params optional by adding a `?` to it.
+Defining what component(s) to load for a given URL is easy and declarative.  
+You can even mix-and-match URL parameters and normal `props`.  
+You can also make params optional by adding a `?` to it.  
+When you add `*` or `+` to param, it matches to match the path to the left of param.  
+`*` matches when `partical_path` is 0 or more characters.  
+`+` matches when `partical_path` is 1 or more characters.
```
Comment (developit) [inline README.md:52]:
```suggestion
Defining what component(s) to load for a given URL is easy and declarative.
Querystring and `:parameter` values are passed to the matched component as props.
 Parameters can be made optional by adding a `?`, or turned into a wildcard match by adding `*` (zero or more characters) or `+` (one or more characters):
```

File: README.md [hunk 2]
```diff
@@ -44,17 +44,22 @@ If there is an error rendering the destination route, a 404 will be displayed.
...
 
 ```js
 <Router>
   <A path="/" />
   <B path="/b" id="42" />
   <C path="/c/:id" />
   <C path="/d/:optional?/:params?" />
-  <D default />
+  <D path="/e/:partical_path*" />
+  <E path="/f/:partical_path+" />
```
Comment (developit) [inline README.md:61]:
```suggestion
  <D path="/e/:remaining_path*" />
  <E path="/f/:remaining_path+" />
```

File: README.md [hunk 3]
```diff
@@ -245,6 +250,50 @@ route('/page-2')  // appends a history entry
 route('/page-3', true)  // replaces the current history entry
 ```
 
+### Nested Route Example
```
Comment (developit) [inline README.md:253]:
```suggestion
### Nested Routers

The `<Router>` is a self-contained component that renders based on the page URL. When nested a Router inside of another Router, the inner Router does not share or observe the outer's URL or matches. Instead, inner routes must include the full path to be matched against the page's URL:
```

File: README.md [hunk 4]
```diff
@@ -245,6 +250,50 @@ route('/page-2')  // appends a history entry
...
+      </Router>
+      <Link activeClassName="active" href="/foo">Foo</Link><br />
+      <Link activeClassName="active" href="/foo/bar">Bar</Link><br />
+      <Link activeClassName="active" href="/foo/baz">Baz</Link>
+    </div>
+  )
+}
+
+render(<App />, document.body)
+```
```
Comment (developit) [inline README.md:295]:
```suggestion
```js
import { h, render } from 'preact';
import Router from 'preact-router';

function Profile(props) {
    // `props.rest` is the rest of the URL after "/profile/"
    return (
      <div>
        <h1>Profile</h1>
        <Router>
          <MyProfile path="/profile/me" />
          <UserProfile path="/profile/:user" />
        </Router>
      </div>
    );
}
const MyProfile = () => (<h2>My Profile</h2>);
const UserProfile = (props) => (<h2>{props.user}</h2>);

function App() {
  return (
    <div>
      <Router>
        <Home path="/" />
        <Profile path="/profile/:rest*" />
      </Router>
      <nav>
        <a href="/">Home</a>
        <a href="/profile/me">My Profile</a>
        <a href="/profile/alice">Alice's Profile</a>
      </nav>
    </div>
  );
}

render(<App />, document.body);
```
```

**Review by developit — COMMENTED** [review]
Hey - thanks for the bump on this, I hadn't seen it. I made some changes that I hope you're okay with!

**Comment by 38elements** [comment]
@developit 
Thank you for reply.
I added your suggestion.

**Review by developit — APPROVED** [review]

**Comment by developit** [comment]
Thanks so much @38elements!

**Pull request merged**